### PR TITLE
feat: datasize module

### PR DESCRIPTION
**Pull request by antmat**
This PR introduces more flexible and powerfull datasize, capable of parsing byte sizes and bit rates and differentiating decimal and binary suffixes (kilo vs kibi and so on).
New datasize is integrated into proto messages providing better marshalling-unmarshalling
@3Hren @sshaman1101 @noxiouz PTAL

### CODE REVIEW

File: cmd/cli/commands/printers.go [hunk 1]
```diff
@@ -139,13 +139,13 @@ func printDeviceList(cmd *cobra.Command, dev *pb.DevicesReply) {
 			}
 		}
 
-		netIn := ds.ByteSize(dev.GetNetwork().GetDevice().GetBandwidthIn()).HR()
-		netOut := ds.ByteSize(dev.GetNetwork().GetDevice().GetBandwidthOut()).HR()
+		netIn := datasize.NewByteSize(dev.GetNetwork().GetDevice().GetBandwidthIn()).HumanReadable()
+		netOut := datasize.NewByteSize(dev.GetNetwork().GetDevice().GetBandwidthOut()).HumanReadable()
```
Comment (sshaman1101) [inline cmd/cli/commands/printers.go]: Looks like `datasize.NewBitRate` will be better here.

File: util/datasize/datasize.go [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+				}
+				*b = bitSize(size) * dim.size
+				return nil
+			default:
+				return errors.New("invalid size for unmarshall, should be float64 or uint64")
+			}
+
+		}
+	}
+	possibleDimensions, _ := json.Marshal(getPossibleDimensions(flags))
```
Comment (3Hren) [inline util/datasize/datasize.go]: Why ignoring errors?

File: util/datasize/datasize.go [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+	return fmt.Errorf(`could not parse bitSize - invalid suffix "%s", possible values are %s`,
+		dimension, possibleDimensions)
+}
+
+func (d *BitRate) UnmarshalText(text []byte) error {
+	dimension, size, err := splitDimension(text)
+	if err != nil {
+		return err
+	}
+	//TODO: Do we really want this kind of behaviour?
```
Comment (3Hren) [inline util/datasize/datasize.go]: No. Raise an error.

Comment (antmat) [reply util/datasize/datasize.go]: discussed to leave this for consistency with ByteSize behaviour

**Pull request merged**